docs/installing-linux-host.md

```diff
@@ -0,0 +1,31 @@
+## Install log
```

Comment:
this log is for a contabo VPS installation. maybe we can write that in the file and filename as well, otherwise we could remove it entirely.

Reply:
renamed the file and it's headline to make it clear it's for contabo.